pkg/integration/tests/commit/commit_with_non_matching_branch.go

```diff
@@ -0,0 +1,47 @@
+package commit
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var CommitWithNonMatchingBranchName = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Commit with defined config commitPrefixes",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig: func(testConfig *config.AppConfig) {
+		testConfig.UserConfig.Git.CommitPrefixes = map[string]config.CommitPrefixConfig{"repo": {Pattern: "^\\w+\\/(\\w+-\\w+).*", Replace: "[$1]: "}}
```

Comment:
I'd prefer this instead:
```go
		testConfig.UserConfig.Git.CommitPrefix = &config.CommitPrefixConfig{
			Pattern: "^\\w+\\/(\\w+-\\w+).*",
			Replace: "[$1]: ",
		}
```

I find it a bit easier to read, and also we're probably going to deprecate CommitPrefixes once we have per-repo config files, and then we don't have to touch this test.